components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -22,38 +23,47 @@ export class ProjectsFilterEffects {
     private requests: ProjectsFilterRequests
   ) { }
 
+  private POLLING_INTERVAL_IN_SECONDS = 120; // 2 minutes
```

Comment:
an arbitrary period i picked. should we poll more/less frequently?

Reply:
🤷‍♂ Seems like that'd be a long time to wait if you were expecting it to show up but I imagine most people are just gonna hit refresh if they want a project to show up right away (even like 30 seconds might be long enough for people to take matters into their own hands). Fine to try this out.

Reply:
I would not go more frequently. Remember this is *not* generally going to be user-makes-a-change then user-sees-different projects. Rather it will be admin-makes-some-changes and sometime later (now within 2 minutes) that will propagate to affected users.

Reply:
yeah i think i'll stick with 2 minutes for now. impatient folks can refresh ;)